lib/FreeRTOS/freertos-plus-mqtt/freertos_mqtt_agent.c

```diff
@@ -375,85 +316,17 @@ static AckInfo_t getAwaitingOperation( MQTTAgentContext_t * pAgentContext,
 
 /*-----------------------------------------------------------*/
 
-#if 0
-
-static bool addSubscription( MQTTAgentContext_t * pAgentContext,
-                             const char * topicFilterString,
-                             uint16_t topicFilterLength,
-                             IncomingPublishCallback_t pIncomingPublishCallback,
-                             void * pIncomingPublishCallbackContext )
+static void removeAwaitingOperation( MQTTAgentContext_t * pAgentContext,
```

Comment:
please declare the function to the top of the file and add documentation about the function similar to the other static function in this file.

Reply:
I had started writing a function to erase based on index rather than packet ID, but I didn't end up using it in this PR. I'll just remove it and add it back later